snap/snapcraft.yaml

```diff
@@ -148,28 +138,27 @@ parts:
     # so then 'pip install PyYAML' gets cross.
     python-packages: [wheel]
     build-environment:
-      - LDFLAGS: -L/usr/lib/python3.8
-      - CPPFLAGS: -I/usr/include/python3.8
+      - LDFLAGS: -L/usr/lib/python3.10
+      - CPPFLAGS: -I/usr/include/python3.10
     override-pull: |
-      # do the usual pull stuff
-      snapcraftctl pull
+      ${SNAP}/libexec/snapcraft/craftctl default
       # Ensure we don't have a dubious ownership error from git with a remote build.
-      git config --global --add safe.directory $SNAPCRAFT_PART_SRC
+      git config --global --add safe.directory $CRAFT_PART_SRC
       # set the version
       version="$(python3 setup.py --version)"
-      snapcraftctl set-version "$version"
+      ${SNAP}/libexec/snapcraft/craftctl set version="${version}"
     override-build: |
-      snapcraftctl build
+      ${SNAP}/libexec/snapcraft/craftctl default
       # why is this needed?
       cp -v completion.bash ../install
       # python3 fixup symlink (snapcraft bug)
-      ln -sf ../usr/bin/python3.8 $SNAPCRAFT_PART_INSTALL/bin/python3
+      ln -sf ../usr/bin/python3.10 $CRAFT_PART_INSTALL/bin/python3
```

Comment:
You can remove this (the plugin in core22 already does this correctly)

Reply:
OK, removed